ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/DivulgenceIT.scala

```diff
@@ -216,10 +216,10 @@ final class DivulgenceIT extends LedgerTestSuite {
     for {
       offer <- alpha.create(proposer, Proposal(from = proposer, to = owner))
       asset <- beta.create(owner, Asset(issuer = owner, owner = owner))
+      _ <- Synchronize.waitForContract(beta, owner, offer)
```

Comment:
I think it'd be easier to read if you import `waitForContract` directly.

Reply:
done